src/psqt.cpp

```diff
@@ -141,8 +141,8 @@ void init() {
           psq[ pc][ s] = v + Bonus[pc][rank_of(s)][f];
           psq[~pc][~s] = -psq[pc][s];
       }
-      inhand[ pc] = v + make_score(50, 50);
-      inhand[~pc] = -inhand[pc];
+
+      inhand[pc] = v;
```

Comment:
For me this line was a bit confusing at first, because I suspected that there will be uninitialized array entries since I had not read the array definition beforehand. Although not required codewise, I think an explicit `type_of` conversion might be helpful for readability.

Reply:
I like your suggestion. Furthermore, the `inhand` array is not behaving in a PSQT-like way so I think it makes sense to take it out of the PSQT namespace altogether.